operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerManager.java

```diff
@@ -141,4 +141,8 @@ public void put(ResourceID key, T resource) {
     getSource(key.getNamespace().orElse(ANY_NAMESPACE_MAP_KEY))
         .ifPresent(c -> c.put(key, resource));
   }
+
+  int numberOfSources() {
+    return sources.size();
```

Comment:
Should we just return an immutable map, this is not very useful outside of the tests.

Reply:
This is indeed just for tests, which is why it's package-protected.